docs/components/table/index.html

```diff
@@ -20,33 +20,35 @@ <h2 id="basic-table">Basic Table</h2>
         <form class="beta-hxForm">
           <fieldset>
             <legend class="beta-hxFieldName">Data Density</legend>
-            <hx-radio-control>
-              <input
-                id="radNormal"
-                name="density"
-                type="radio"
-                v-model="isCondensed"
-                :value="false"
-              />
-              <label for="radNormal">
-                <hx-radio></hx-radio>
-                Normal
-              </label>
-            </hx-radio-control>
+            <hx-radio-set>
+              <hx-radio-control>
+                <input
+                  id="radNormal"
+                  name="density"
+                  type="radio"
+                  v-model="isCondensed"
+                  :value="false"
+                />
+                <label for="radNormal">
+                  <hx-radio></hx-radio>
+                  Normal
+                </label>
+              </hx-radio-control>
 
-            <hx-radio-control>
-              <input
-                id="radCondensed"
-                name="density"
-                type="radio"
-                v-model="isCondensed"
-                :value="true"
-              />
-              <label for="radCondensed">
-                <hx-radio></hx-radio>
-                Condensed
-              </label>
-            </hx-radio-control>
+              <hx-radio-control>
+                <input
+                  id="radCondensed"
+                  name="density"
+                  type="radio"
+                  v-model="isCondensed"
+                  :value="true"
```

Comment:
can we pls move `:value="true"` above `id="radCondensed"`? seems like we're going with this pattern in this p/r

Reply:
moved `:value="true"` above `id="radCondensed"`